superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -384,7 +387,7 @@ class Header extends React.PureComponent {
 
         <div className="button-container">
           {userCanSaveAs && (
-            <div className="button-container">
+            <div className="button-container" data-test="button-container">
```

Comment:
```suggestion
            <div className="button-container" data-test="dashboard-edit-actions">
```
Maybe something a bit more explicit like so?

Reply:
Yes it seems to be more reasonable

Reply:
I applied this suggestion! thanks!